packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -9,6 +9,7 @@ import '../application_package.dart';
 import '../base/file_system.dart';
 import '../base/io.dart';
 import '../base/os.dart';
+import '../base/platform.dart' as p;
```

Comment:
Otherwise it collides with `TargetPlatform get platform`, which is why I added the TODO to change that to `TargetPlatform get targetPlatform`.  I'll remove the alias once I follow-up on that TODO.